utils/R/read.output.R

```diff
@@ -87,10 +87,11 @@ model2netcdf <- function(runid, outdir, model, lat, lon, start_date, end_date) {
 ##' @param start.year first year of output to read (should be greater than ) 
 ##' @param end.year last year of output to read
 ##' @param variables variables to be read from model output
+##' @param dataframe A boolean that will return output in a data.frame format with posix column. Usefull for align.data and plotting. 
```

Comment:
I totally take this back. It seems to be used by enough functions that I'm getting nervous that I could catch every instance. Also defaulting to false means the default is a netCDF file, which feels more flexible than a dataframe.